browser/ui/webui/settings/brave_settings_localized_strings_provider.cc

```diff
@@ -1067,6 +1067,71 @@ void BraveAddSyncStrings(content::WebUIDataSource* html_source) {
                          passphraseDecryptionErrorMessage);
 }
 
+void BraveAddEmailAliasesStrings(content::WebUIDataSource* html_source) {
+  webui::LocalizedString localized_strings[] = {
```

Comment:
shouldn't this return early and not add anything if the feature is disabled?

Reply:
yes, fixed